tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh

```diff
@@ -129,7 +129,9 @@ for fs in "$POOL" "$POOL/pclone" "$POOL/$FS" "$POOL/$FS/fs1" \
 	rand_set_prop $fs dnodesize "legacy" "auto" "1k" "2k" "4k" "8k" "16k"
 	rand_set_prop $fs setuid "on" "off"
 	rand_set_prop $fs snapdir "hidden" "visible"
-	rand_set_prop $fs xattr "on" "off"
+	if ! is_freebsd; then
+		rand_set_prop $fs xattr "on" "off"
+	fi
```

Comment:
I thought `xattr` existed as a property on all implementations, only the "sa" and "dir" values were only supported on Linux.  Is that not the case?

Reply:
The property is visible but can't be changed.